cmd/kubeapps-apis/plugins/helm/packages/v1alpha1/server_test.go

```diff
@@ -199,33 +199,80 @@ func TestIsValidChart(t *testing.T) {
 		expected bool
 	}{
 		{
-			name:     "it returns true if the chart is correct",
-			in:       chartOK,
+			name: "it returns true if the chart name, ID, repo and versions are specified",
+			in: &models.Chart{
+				Name: "foo",
+				ID:   "foo/bar",
+				Repo: &models.Repo{
+					Name:      "bar",
+					Namespace: "my-ns",
+				},
+				ChartVersions: []models.ChartVersion{
+					{
+						Version: "3.0.0",
+					},
+				},
+			},
```

Comment:
I *think* this is more than a personal preference: unit tests are the one place where DRY is not always better. Having to scroll up and down the file to find what `chartOK` is reduces the readability of the tests. So throughout the tests I was looking at, I've updated with more verbose but hopefully more obvious input/output.

Reply:
Mmm, I see... yep, I kind of agree with it, but still not 100% convinced :-P
The point is: what if we need and use the "concept of a perfect chart model" across different tests? Do we really need to write it again and again? It could be more readable, that's for sure... but, when one has to perform a change.. it tedious to replicate it over every test suite.

However, no strong opinion whatsoever. Let's go with the verbose way for consistency.

Reply:
What I've done in the past in that situation, when you begin to have the same large data struct in lots of places, is to instead define a function like `createValidChart(options...)` which can be called without options, or specific options to set specific things. Not sure if that helps here, but I'll take a look in the next PR and we can discuss there.